cegs_portal/search/views/v1/dna_features.py

```diff
@@ -106,6 +107,9 @@ def sort_key(feature_reo):
             first_child = first_feature.children.first()
             child_feature_type = first_child.get_feature_type_display()
 
+        first_feature = feature_reos[0][0]
```

Comment:
Instead of writing `first_feature = feature_reos[0][0]` multiple times you could write it once higher up.

Reply:
Completed